ic-os/guestos/context/docker-base.prod

```diff
@@ -1 +1 @@
-ghcr.io/dfinity/guestos-base@sha256:0a485d2d033df5aa7efd4ccbaad432c29d18f0f165cdcbeed5d49790c0d16340
+ghcr.io/dfinity/guestos-base@sha256:682e502bad88871a0d37ae8892880080a4c5d5360943ab40d0d8dce851d8eb4e
```

Comment:
This contains changes for the GuestOS 24.04 upgrade, we'll keep this in an RC for the initial rollout